integration-cli/docker_cli_cp_test.go

```diff
@@ -472,7 +472,7 @@ func TestCpVolumePath(t *testing.T) {
 	}
 
 	cleanedContainerID := strings.TrimSpace(out)
-	defer deleteContainer(cleanedContainerID)
+	defer dockerCmd(t, "rm", "-fv", cleanedContainerID)
```

Comment:
oh, I know why!
Dead containers happen when `rm` failed, but when you `rm -f` them again it forcibly removes them from docker regardless if there is an issue.

So first time with `rm -f` it had an issue, is marked as dead.  If you just `rm` (no `-f`) it won't remove if the issue isn't resolved. But when you `rm -f` again, it will remove from docker regardless of issues.